README.md

```diff
@@ -117,7 +117,8 @@ A full example could be:
           formats : "QR_CODE,PDF_417", // default: all but PDF_417 and RSS_EXPANDED
           orientation : "landscape", // Android only (portrait|landscape), default unset so it rotates with the device
           disableAnimations : true, // iOS
-          disableSuccessBeep: false // iOS and Android
+          disableSuccessBeep: false, // iOS and Android
+          continuosMode: false // Android
```

Comment:
Can you switch this to `continuousMode` to keep the options consistent with English spelling?